Mathlib/CategoryTheory/Monoidal/Comon_.lean

```diff
@@ -0,0 +1,173 @@
+/-
+Copyright (c) 2024 Lean FRO LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import Mathlib.CategoryTheory.Monoidal.CoherenceLemmas
+import Mathlib.CategoryTheory.Limits.Shapes.Terminal
+
+/-!
+# The category of comonoids in a monoidal category.
+
+We define comonoids in a monoidal category `C`.
+
+## TODO
+* `Comon_ C ≌ Mon_ (Cᵒᵖ)`
+* An oplax monoidal functor takes comonoid objects to comonoid objects.
+  That is, a oplax monoidal functor `F : C ⥤ D` induces a functor `Comon_ C ⥤ Comon_ D`.
+* Comonoid objects in `C` are "just"
+  oplax monoidal functors from the trivial monoidal category to `C`.
+* The category of comonoids in a braided monoidal category is monoidal.
+  (It may suffice to transfer this across the equivalent to monoids in the opposite category.)
+
+-/
+
+set_option linter.uppercaseLean3 false
```

Comment:
I assume that this is a copy-paste leftofer from another file that's not needed here?